app/services/albums.js

```diff
@@ -0,0 +1,28 @@
+const request = require('request-promise-native'),
+  errors = require('../errors'),
+  { endpoints } = require('../constants/endpoints'),
+  logger = require('../logger/');
+
+errors.ALBUM_API_ERROR = 501;
```

Comment:
Move this to the `middleware/errors.js` file. Add this in the `statusCodes` object.